docs_checker/check.scala

```diff
@@ -58,68 +59,118 @@ def parse(content: Seq[String], currentCommands: Seq[Commands], context: Context
 
 case class TestCase(path: Path, failure: Option[Throwable])
 
-def checkPath(path: Path): Seq[TestCase] =
+def checkPath(dest: Option[Path])(path: Path): Seq[TestCase] =
   try
     if !Files.isDirectory(path) then
       if path.getFileName.toString.endsWith(".md") then
-        checkFile(path)
+        checkFile(path, dest)
         Seq(TestCase(path, None))
       else Nil
     else
       val toCheck =
         Files.list(path).iterator.asScala.filterNot(_.getFileName.toString.startsWith("."))
-      toCheck.toList.flatMap(checkPath)
+      toCheck.toList.flatMap(checkPath(dest))
   catch
     case e: Throwable =>
       e.printStackTrace()
       Seq(TestCase(path, Some(e)))
 
-def checkFile(file: Path) =
+def ammPath(p: Path) = os.Path(p.toAbsolutePath)
+
+val fakeLineMarker = "//fakeline"
+
+def checkFile(file: Path, dest: Option[Path]) =
   val content  = Files.lines(file).iterator.asScala.toList
```

Comment:
```suggestion
  val content  = os.read.lines(file)
```
but, you have to replace `Path` to `os.Path`